package.json

```diff
@@ -7,7 +7,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "mkdir": "mkdir -p build",
     "build": "npm run clean && npm run mkdir && npm run build:html && npm run build:css",
-    "watch": "npm run watch:html & npm run wathc:css",
+    "watch": "npm run watch:html & npm run watch:css",
```

Comment:
A small spelling error here prevents the `watch:css` task from running, so i wouldn't build your css on file change.